exercise/viewbase.py

```diff
@@ -122,6 +124,12 @@ def get_summary_submissions(self, profile=None):
         self.submissions = self.summary.get_submissions()
         self.note("summary", "submissions")
 
+    def get_cached_points(self, profile=None):
+        cache = CachedPoints(self.instance, profile or self.request.user, self.content, self.is_course_staff)
```

Comment:
Yes, I tested that the teacher sees the student's deviations when viewing a submission or the "user results" page.

Reply:
> Based on SubmissionMixin.get_cached_points, profile is actually a User, but the variable should then be named user.

Yes, and looks like I also forgot to add type hints to the methods I edited. I added them now.

The parameter was called `profile` because I had copied the `get_summary_submissions` method's signature, where the parameter was also called `profile` even though it is of type `User`. I've now fixed the parameter's name in both of these methods.